Makefile

```
@@ -7,32 +7,45 @@
 GOBIN = ./build/bin
 GO ?= latest
 GORUN = go run
+PROJECT_NAME = $(shell git remote get-url origin | xargs basename -s .git)
```

```suggestion
PROJECT_NAME = "go-ethereum"
```
Let's just keep it simple. Or even simpler: remove this and inline it here:
```
@echo " Choose a command run in go-ethereum:"
```